docker-compose.yml

```diff
@@ -35,21 +35,14 @@ services:
         build:
             context: .
             dockerfile: docker/Dockerfile-caddy
+        env_file: .env
         restart: unless-stopped
         ports:
-          - 80:80
-          - 443:443
+          - 2015:2015
```

Comment:
Why are we setting ports to `2015`? Is that caddy specific?

Reply:
Yes, that's the default port for Caddy. When it's running on localhost, Caddy defaults to port 2015 and serves HTTP only. When it's running on a real website, Caddy defaults to port 80 (for HTTP) and 443 (for HTTPS). We expose 80 and 443 in the docker-compose override file, when we deploy the official site.